vispy/gloo/glir.py

```diff
@@ -259,19 +260,25 @@ def as_es2_command(command):
 class BaseGlirParser(object):
     """ Base clas for GLIR parsers that can be attached to a GLIR queue.
     """
-    
+
+    def __init__(self):
+        self.capabilities = dict(
```

Comment:
Would it make sense for this to be an attribute of the context instead of the parser?